code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -39,7 +39,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	return ..()
 
 /datum/reagent/consumable/ethanol/on_mob_life(mob/living/carbon/C)
-	if(C.drunkenness < volume * boozepwr * ALCOHOL_THRESHOLD_MODIFIER || boozepwr < 0)
+	if(C.drunkenness < volume * boozepwr * ALCOHOL_THRESHOLD_MODIFIER * 0.25 || boozepwr < 0)
```

Comment:
Simply alter `ALCOHOL_THRESHOLD_MODIFIER` to be 0.25 instead of 1. It is, from a search through the repo, only used here.

Reply:
True, Will fix.